src/tests/dictionaryConcepts/components/CreateMapping.test.js

```diff
@@ -74,4 +74,183 @@ describe('Test suite for dictionary concepts components', () => {
     const inputs = wrapper.find('select');
     expect(inputs).toHaveLength(2);
   });
+
+  it('should test update new mappings for new rows', () => {
```

Comment:
What are you testing about new mappings for new rows?

Reply:
Secondly, this being a test, aren't do we need to include the word test?

Reply:
The same applies everywhere below

Reply:
I have updated the test descriptions please review.